src/frontend/components/SiteLayout/index.tsx

```diff
@@ -101,6 +117,20 @@ export const SiteLayout: React.FC = ({ children }) => {
                         </SidebarLink>
                       ))}
                   </div>
+                  <Button
+                    onClick={redirectTo('/account/logout/')}
+                    style={{ padding: 0 }}
+                  >
+                    <Box
+                      direction="row"
+                      pad={{ horizontal: 'medium', vertical: 'small' }}
+                      gap="small"
+                      align="center"
+                    >
+                      <Logout />
+                      <Text>{intl.formatMessage(messages.logOutBtn)}</Text>
+                    </Box>
+                  </Button>
```

Comment:
It seems you can use something like https://storybook.grommet.io/?path=/story/controls-button-icon-label--icon-label

```suggestion
                  <Button
                    onClick={redirectTo('/account/logout/')}
                    icon={<Logout />}
                    label={intl.formatMessage(messages.logOutBtn)}
                  />
```